guides/common/modules/con_introduction-to-application-centric-deployment.adoc

```diff
@@ -14,13 +14,13 @@ You can scale an application instance vertically via compute profile or horizont
 Within ACD, Ansible playbooks are executed on {ProjectServer}.
 Ensure that {ProjectServer} can connect to your hosts using Ansible.
 
-.Host Centric vs. Application Centric Deployment
+.Host centric vs. application centric deployment
```

Comment:
Should we add this to non-caps list?

Reply:
I noticed that they were capitalized in other places as well. 

Re: hyphens. In theory, yes. But making the change globally should be done in a separate PR.